iothub/device/samples/convention-based-samples/readme.md

```diff
@@ -12,41 +12,557 @@ products:
 urlFragment: azure-iot-pnp-device-samples-for-csharp-net
 ---
 
-# IoT Plug And Play device samples
+# IoT Plug And Play (PnP) device/ module APIs
 
-These samples demonstrate how a device that follows the [IoT Plug and Play conventions][pnp-convention] interacts with IoT Hub or IoT Central, to:
+Device(s)/ module(s) connecting to IoT Hub that announce their DTDL model Id during initialization can now perform convention-based operations. One such convention supported is [IoT Plug and Play][pnp-convention].
 
-- Send telemetry.
-- Update read-only and read-write properties.
-- Respond to command invocation.
+These device(s)/ module(s) can now use the native PnP APIs in the Azure IoT device SDKs to directly exchange messages with an IoT Hub, without having to manually format these messages to follow the PnP convention.
```

Comment:
Since you used 'These', I would expect plural to always follow. So maybe no need for the (s) on device and module.